openssl-dynamic/src/main/c/ssl_private.h

```diff
@@ -199,6 +199,11 @@ extern void *SSL_temp_keys[SSL_TMP_KEY_MAX];
 #define SSL_CTRL_GET_CLIENT_CERT_TYPES          103
 #endif
 
+#ifndef SSL_ERROR_WANT_CERTIFICATE_VERIFY
+// See https://github.com/google/boringssl/blob/chromium-stable/include/openssl/ssl.h#L538
+#define SSL_ERROR_WANT_CERTIFICATE_VERIFY 16
```

Comment:
I'm guessing this is defined just to ease compilation with OpenSSL or something. You might want to use -1 or something that'll never happen. Otherwise you might run into issues later on if OpenSSL allocates 16 for some other status.

Reply:
Good point

Reply:
fixed